dashboard/app/controllers/api/v1/sections_controller.rb

```diff
@@ -6,7 +6,7 @@ class Api::V1::SectionsController < Api::V1::JsonApiController
   # GET /api/v1/sections
   # Get the set of sections owned by the current user
   def index
-    render json: @sections.map(&:summarize)
+    render json: current_user.sections.map(&:summarize)
```

Comment:
New approach: Instead of messing with admin permissions in ability.rb (although I still think there might be merit to that generally) we fix the implementation of our controller to accurately return "sections owned by the current user," rather than "sections the current user has permission to index" which is what leaning on CanCan(Can) `load_and_authorize_resource` was doing.